recipe/meta.yaml

```diff
@@ -26,13 +25,13 @@ requirements:
     - cairo 1.14.*
     - flex
     - gettext  # [osx]
-    - glib 2.51.*
+    - glib 2.55.*
     - libffi 3.2.*
     - pkg-config  # [not win]
     - python
     - toolchain
   run:
-    - glib 2.51.*
+    - glib 2.55.*
```

Comment:
Any reason this doesn't match up with the [`glib 2.51` global pinning]( https://github.com/conda-forge/conda-forge.github.io/blob/cbbe486d281aec41c69ca96d09acc4813fa96abb/scripts/pin_the_slow_way.py#L53 )?